bindings/experimental/distrdf/python/DistRDF/Operation.py

```diff
@@ -102,7 +79,7 @@ def is_action(self):
         Returns:
             bool: True if the current operation is an action, False otherwise.
         """
-        return self.op_type == Operation.ACTION
+        return self.op_type == "ACTION"
```

Comment:
Why converting these back into string literals instead of string constants?

Reply:
Anyway, I don't know if evaluating a string literal is much slower than the lookup to the module-level (or enum-scope) equivalent constant

Reply:
I think an enum would be nicer :)